device/nokia/x86_64-nokia_ixr7250e_sup-r0/platform_ndk.json

```diff
@@ -31,6 +31,26 @@
     {
       "key": "sonic_log_level",
       "stringval": "debug"
+    },
+    {
+      "key": "thermal_low_margin_threshold",
+      "intval": 10
+    },
+    {
+      "key": "thermal_log_current_threshold",
+      "intval": 3
+    },
+    {
+      "key": "thermal_log_margin_threshold",
+      "intval": 3
+    },
+    {
+      "key": "thermal_log_min_threshold",
+      "intval": 5
+    },
+    {
+      "key": "thermal_log_max_threshold",
+      "intval": 1
     }
```

Comment:
In the **class PublishThermalAlgoAction** in chassis/sonic_platform/thermal_actions.py we do a **return** in execute() function. 

Is there a new different service -- I assume the new sensord implementation which will be doing a UpdateTempInfoPb()?

Reply:
Yes, the thermal handling is done behind the scenes at our NDK.  Temp info is still updated at Redis DB, but we publish to our Supervisor card directly via NDK mechanism for input to fan control algorithm.